source/tutorial/kerberos.txt

```diff
@@ -68,12 +70,24 @@ Kerberos authentication on either Windows or macOS.
                setspn.exe -A mongosql/BI.EXAMPLE.COM mongosql
 
             Open the `Active Directory Administrative Center <https://docs.microsoft.com/en-us/windows-server/identity/ad-ds/get-started/adac/active-directory-administrative-center>`__
-            and double-click on your MongoDB service user 
+            and double-click on your MongoDB service user
             (``mongodb``).
 
-            Set the ``mongosql`` user to delegate for the 
+            Set the ``mongosql`` user to delegate for the
             ``mongodb`` user from the ``BI.EXAMPLE.COM`` host.
 
+
+            .. admonition:: Linux Schema User Authenticating to ADC
```

Comment:
Maybe Schema User Authenticating to ADC from a Linux Host?

Reply:
Another alternate title: **ADC Authentication from a Linux Host**